lib/widgets/inbox.dart

```diff
@@ -491,10 +518,25 @@ class _TopicItem extends StatelessWidget {
                   overflow: TextOverflow.ellipsis,
                   topic))),
               const SizedBox(width: 12),
-              // TODO(#384) show @-mention indicator when it applies
+              if (isMentioned) const _AtMentionMarker(),
               Padding(padding: const EdgeInsetsDirectional.only(end: 16),
                 child: UnreadCountBadge(backgroundColor: subscription.colorSwatch(),
                   count: count)),
             ])))));
   }
 }
+
+class _AtMentionMarker extends StatelessWidget {
```

Comment:
These two commits are best squashed together:
ce34ec011 inbox: Show at-mention marker on topics
b6ac52216 inbox [nfc]: Pull out _AtMentionMarker

because the latter basically amounts to rethinking how to arrange some of the code added by the former, and so it's simplest to go straight to the revised way of arranging it.

Reply:
In fact let's squash those together with the next commit, handling streams:
ce34ec011 inbox: Show at-mention marker on topics
b6ac52216 inbox [nfc]: Pull out _AtMentionMarker
6e74d2a43 inbox: Show at-mention marker on streams

That way there isn't an intermediate state where we have @-mention markers on the individual topics but missing on the streams, which feels a little glitchy given expectations set by web. And conversely the squashed commit is still a quite reasonable size.